CHANGES.rst

```diff
@@ -1,6 +1,24 @@
 Release Notes
 =============
 
+1.6.2
+-----
+
+* Support de-vendoring for installs. (#666)
+  `PR #666 <https://github.com/pantsbuild/pex/pull/666>`_
+
+* Add User-Agent header when resolving via urllib (#663)
+  `PR #663 <https://github.com/pantsbuild/pex/pull/663>`_
+
+* Fix interpreter finding (#662)
```

Comment:
We aren’t allowed to rename this here, are we? I realized right after hitting merge I had wanted to rename the commit message to something like “Fix interpreter failing to find python{2,3} bin name”. It was too late to fix once merged into master. 

Would be great to fix here at least, but not sure we want to diverge commit message from release notes.

Reply:
There is no policy I'm aware of for pants or pex here, but I've been sticking to damage done via commit.